tensorboard/plugins/debugger/tf_debugger_dashboard/tf-tensor-value-multi-view.html

```diff
@@ -48,6 +48,13 @@
           value: null,
         },
 
+        // Callback invoked when a tensor name is clicked in one of the tensor
+        // value views. Set externally.
+        tensorNameClicked: {
```

Comment:
If you would like this component to delay initialization til this property is defined, you can just leave out the value, ie `tensorNameClicked: Object`.

Reply:
Done. Thanks for the suggestion.